src/SSHLibrary/javaclient.py

```diff
@@ -104,6 +104,7 @@ class SFTPClient(AbstractSFTPClient):
     def __init__(self, ssh_client, encoding):
         self._client = SFTPv3Client(ssh_client)
         self._client.setCharset(encoding)
+        self._encoding = encoding
```

Comment:
What is this related to?

Reply:
I have removed this to AbstractSFTPClient.

It is used a lot of place. for example:
```
class SFTPClient(AbstractSFTPClient):

    def __init__(self, ssh_client, encoding):
        self._client = ssh_client.open_sftp()
        super(SFTPClient, self).__init__(encoding)

    def _list(self, path):
        path = path.encode(self._encoding)
        for item in self._client.listdir_attr(path):
            filename = item.filename
            if is_bytes(filename):
                filename = filename.decode(self._encoding)
            yield SFTPFileInfo(filename, item.st_mode)

    def _stat(self, path):
        path = path.encode(self._encoding)
        attributes = self._client.stat(path)
        return SFTPFileInfo('', attributes.st_mode)
```